cypress/e2e/patient_spec/patient_registration.cy.ts

```diff
@@ -138,6 +138,66 @@ describe("Patient Creation with consultation", () => {
     patientPage.verifyPatientNameList(patientOneName);
   });
 
+  it("Create a new patient with all field in registration form with age and verify", () => {
```

Comment:
delete this entire test and modify exist any two cypress test to verify this

Reply:
@nihal467 I cant exactly find a way to replace this test with other , can you give some guidance on how to do ?